app/resources/create.html.template

```diff
@@ -94,6 +115,14 @@
               </div>
             {% endif %}
 
+            <a href="#" class="identify-button" style="text-decoration:none">
+              {% trans "+ Other personal identify detail" %} <br>
```

Comment:
This is a bit weird as English. How about e.g.,

Input more identifying information
Input more information to identify the person
More identifying information

Rename other button labels accordingly.